### ISSUE TITLE
Send notifications on different actions performed in the system

### ISSUE THREAD

**Issue by TCMeldrum** [UserOfficeProject/user-office-project-issue-tracker#983]
### What is the problem and why is it a problem
It would be useful to send notifications at various times, such as proposal assigned to reviewer, role assigned to panellist, review in saved state for X days.

### Steps to reproduce (if it's a bug).
N/A

**Comment by srconway** [comment]
Clarify, what notifications are sent currently?

**Comment by TCMeldrum** [comment]
Currently, we can on any status change set a up a email to be sent to any of: 

-   PI
-   CO_PROPOSERS
-   INSTRUMENT_SCIENTISTS
-   FAP_REVIEWERS
-   OTHER - hard coded email

I think we need to know what events we would like to send emails on and too who.

**Comment by TCMeldrum** [comment]
I think we might only send emails on proposal submitted and call created

**Comment by srconway** [comment]
@Cat-Lucifer Do you know what emails we currently send in relation to the current scoring portal and what emails they want us to send in the new system, and to whom.

**Comment by Cat-Lucifer** [comment]
@srconway  There are no notifications set up in the SharePoint scoring portal.  I *think* that this was referring to a notification sent to reviewers when they had something to review and various notifications sent to FAP Secs (e.g. technical review done, scientific review done, etc.).

@TCMeldrum I wasn't at the FAP Sec demo on Friday, but I understand notifications was mentioned.  Is there any more clarification from that?

**Comment by TCMeldrum** [comment]
@Cat-Lucifer The consensus was that some would be nice, but there was no agreement on what. Not sure what the best way to get this information might be,

**Comment by TCMeldrum** [comment]
Added sending emails to chair and sec here: https://github.com/UserOfficeProject/user-office-core/pull/527

**Comment by joshhdawes** [comment]
@Cat-Lucifer could this be a requirement for the next round?

**Comment by Cat-Lucifer** [comment]
@joshhdawes  Possibly... Let's see what comes up in this week's feedback review meeting and take this one from there.

**Comment by joshhdawes** [comment]
@Cat-Lucifer I don't think this one came up in the feedback review meeting - any thoughts now that we've been through all the feedback?

**Comment by Cat-Lucifer** [comment]
@joshhdawes  This is one to be discussed with GLs at some point - it is related to the issue on FAP Sec notifications.

**Comment by joshhdawes** [comment]
@Cat-Lucifer is this issue okay to be done as part of the FAP meetings work (and therefore possibly delayed)?

Just thought I'd check since it looks like some of the statuses mentioned in this issue might be relevant to parts of the system other than meetings.

**Comment by Cat-Lucifer** [comment]
@joshhdawes Yep, this is fine.
